log4go_test.go

```diff
@@ -57,6 +57,7 @@ var formatTests = []struct {
 		Formats: map[string]string{
 			// TODO(kevlar): How can I do this so it'll work outside of PST?
 			FORMAT_DEFAULT: "[2009/02/13 23:31:30 UTC] [EROR] (source) message\n",
+			FORMAT_MILLIS:  "[2009/02/13 23:31:30.123] [EROR] (source) message\n",
```

Comment:
I believe the request was to match the log4j formats, so if that's the case we should match them and disregard the TZ.